src/adapters/Cornerstone/CircleRoi.js

```diff
@@ -29,7 +28,9 @@ class CircleRoi {
             toolType: CircleRoi.toolType,
             active: false,
             cachedStats: {
-                area: NUMGroup.MeasuredValueSequence.NumericValue
+                area: NUMGroup.MeasuredValueSequence.NumericValue,
+                radius: 5,
```

Comment:
Needed dummy values here - eventually should use the right set of values read from the response.

Reply:
Changed to be 0 @sedghi  - added a comment to the code explaining why it has the value 0 as it is jsut a dummy.